.github/workflows/deploy.workflow.yml

```
@@ -0,0 +1,43 @@
+name: build and deploy
+
+on:
+  # run when action is manually triggered
+  workflow_dispatch:
+  # run on pushes to master branch
+  push:
+    branches: [ master ]
+  # run daily at 08:00am UTC (on the default or base branch)
+  # https://crontab.guru/#0_8_*_*_*
+  schedule:
+    - cron: '0 8 * * *'
+
+jobs:
+  # job id/key
+  build-and-deploy:
+    # job name
+    name: Build and Deploy
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+
+      - run: npm ci
+      - run: npm run generate
+
+      - name: Deploy
+        run: cd deploy && ./deploy_cos.sh
+        shell: bash
+        env:
+          AIRTABLE_API_KEY: ${{ secrets.AIRTABLE_API_KEY }}
+          encrypted_rclone_iv: ${{ secrets.encrypted_rclone_iv }}
+          encrypted_rclone_key: ${{ secrets.encrypted_rclone_key }}
```

Secrets names were registered in all-uppercase, although in the script, it uses snake_case names. Let's match each case:
```suggestion
          encrypted_rclone_iv: ${{ secrets.ENCRYPTED_RCLONE_IV }}
          encrypted_rclone_key: ${{ secrets.ENCRYPTED_RCLONE_KEY }}
```

---

It doesn't matter they're case insensitive matches. The all caps is forced by the github ui (I didn't actually enter that in). This will work fine as is.